### PR TITLE
Add Free Angular 2 Course To Classes

### PR DESCRIPTION
**Pull request by nickkenens**
Added a Free Udemy Course that will teach people the fundamentals of Angular 2.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,6 +162,7 @@
 * [How to Import Immutable.js](http://stackoverflow.com/questions/29527024/how-do-i-import-immutable-js-into-angular-2-alpha)
 
 ### Classes
+* [Free Udemy Course - Angular 2 Fundamentals](https://www.udemy.com/angular-2-fundamentals)
```
Comment (diraulo) [inline README.md]: Hi @nickkenens The course doesn't appear to be free :confused:

**Comment by crissdev** [comment]
I really don't see the point of adding such links. A quick search for Angular courses will get anyone to any of the sites having paid or free courses. 
So let's better make technology known instead of _people_.

File: README.md (hunk 1 above)
Comment (crissdev) [inline README.md]: @diraulo  :+1:

**Comment by timjacobi** [comment]
Thanks for the hint guys. I do check the links from time to time so expect this to be removed on the next interation.

**Pull request merged**